src/dotnet-interactive/CommandLine/CommandLineParser.cs

```diff
@@ -478,6 +478,7 @@ private static IServiceCollection RegisterKernelInServiceCollection(IServiceColl
                          .UseDefaultMagicCommands()
                          .UseLog()
                          .UseAbout()
+                         .UseProxyKernelWithSignalR()
```

Comment:
Can we come up with better naming than `UseProxy*`?

Reply:
Proposing

```
.UseConnectionOverNamedPipe()
.UseConnectionOverSignalR()
```

Reply:
For each transport we have two APIs, one to expose a kernel as a server, and the other to snap in clients as subkernels. How can we name them so it's obvious which end of the connection we're talking about?